src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java

```diff
@@ -68,11 +92,15 @@
  * try {
  *   throw new RuntimeException();
  * } catch (RuntimeException expected) {
- * }
+ * } // ok
  * try {
  *   throw new RuntimeException();
  * } catch (RuntimeException ignore) {
- * }
+ * } // ok
```

Comment:
I do not see comment "no handling" comment.

please make all example to be absolutely the same with difference only in trailing comments "//ok" "//violation"
please do copy-paste and change only traililng comments from `// ok ` -> `// violation` or `// violation` -> `// ok`.
NO other changes.

Reply:
okay sure,I actually did that so that users can understand that it is correct not because of `// no handling` but because of `ignore` variable

Reply:
code should be the same, SAME, IDENTICAL.
difference is only trailing comments.
user should see
```
try {
  throw new RuntimeException();
} catch (RuntimeException ex) {
  // This is expected
}
```
in first and second examples